.github/workflows/release.yml

```
@@ -138,18 +138,39 @@ jobs:
             openssl dgst -sha256 -sign <(printf "%s" "$OPENSSL_KEY") -out "${archive}.signature" "${archive}"
           done
 
+      - name: Checkout the repo
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          path: cli
+
       - name: Create GitHub release
         id: create_release
         uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6.4.1
         with:
           # The response is explicitly returned here so it will be available for other steps
           script: |
+            const fs = require("node:fs");
+
+            const tag = process.env.GITHUB_REF_NAME;
+            const prerelease = tag.includes("-rc");
```

```suggestion
            const prerelease = tag.includes("rc");
```
The other checks for a pre-release in this file do not include the hyphen/dash/whatever.